docs/Microservices/MPS/securityMPS.md

```diff
@@ -15,38 +14,38 @@ In addition to the above assets, there are best practices that are recommended t
+### 1. Intel&reg; AMT remote admin credentials
+Intel&reg; AMT remote admin credentials allows a user to remotely control an Intel&reg; AMT device and these credentials are configured in AMT Firmware by a configuration server (ex. RPS). When a user performs an action on the device using REST API, MPS fetches the corresponding credentials of that device from the configured secrets store (ex. Vault) and uses it as part of digest authentication. It is highly recommended to use a strong password (as per AMT password requirements) that is unique per device to make it more secure.
 
-### 2. Intel&reg; MPS credentials
-Every Intel&reg; device needs to be authenticated prior to successful connection to MPS. MPS credentials are used to authenticate every Intel&reg; device. After establishing TLS connection with MPS, device sends it's username and password, which will then be used to authenticate. It is highly recommended that every device use a unique username and password.
+### 2. Intel&reg; AMT CIRA credentials
+When an Intel&reg; device attempts to establish a connection to the MPS, there are two checks that are performed by MPS prior to allowing the connection.  
+1. The Intel&reg; AMT device's GUID must be listed in the MPS database.  This is typically added by using the Devices POST API.  
+2. The Intel&reg; AMT device needs to supply the correct credentials to MPS.  These credentials are checked against the username (stored in the database) and password (stored in Vault).  It is highly recommended that every device use a unique password.
 
-### 3. Device Allowlist (List of GUIDs)
-Each Intel® AMT device has a unique identifier (GUID) assigned to it and this GUID is assigned by Original Equipment Manufacturer (OEM). As discussed above, devices are authenticated by MPS using MPS credentials. But even before validating credentials, only allowlist Intel® AMT devices can be allowed to connect to make it more secure and this is being done using the allowlist GUID.  When Vault is being used to store device credentials, this list of devices is also the allowlist.
+### 3. Authorize API end point
+MPS has a default Authorize API end point for issuing a JWT for basic user authentication to the REST APIs.  This is in place so developers can easily start using the Open AMT Cloud Toolkit microservices without needing to setup a full user authentication service for evaluation of the software.  In production, it is recommended to use a robust user authentation service (ex. OAuth2, OpenID, LDAP, etc) and disable this Authorize API end point by leaving the MPS Web User credentials blank in the MPS configuration file.
```

Comment:
MPS supports basic user authentication to the REST APIs with an Authorize API endpoint for issuing a JWT. This eliminates the need to set up a full user authentication service before evaluating the software and enables developers to use the microservices right away. 

However, in a production deployment,  use a robust user authentication service (e.g., OAuth2, OpenID, or LDAP) and disable the Authorize API endpoint by leaving the MPS Web User credentials blank in the MPS configuration file.